eng/pipelines/templates/steps/cmake-build.yml

```diff
@@ -2,15 +2,17 @@ parameters:
   BuildArgs: ''
 
 steps:
-  - task: CMake@1
-    inputs:
-      cmakeArgs: --version
-    displayName: cmake version
-  - task: CMake@1
-    inputs:
-      cmakeArgs: ${{ parameters.BuildArgs }} ..
+  - script: mkdir build
```

Comment:
Are you sure mkdir works cross-platform?

Reply:
It does work cross platform: 

Windows - [docs](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/mkdir), [run](https://dev.azure.com/azure-sdk/public/_build/results?buildId=411265&view=logs&j=ea3cb55d-7776-5c6b-1bf6-a444b419cb36&t=84122ea0-8743-50eb-cd5b-c982519ba8f7)

Bash - [docs](https://ss64.com/bash/mkdir.html), [run](https://dev.azure.com/azure-sdk/public/_build/results?buildId=411265&view=logs&j=f578152f-9f12-52ef-8faa-e76d6b953f78&t=542506c0-5ebe-587f-ef76-8d250f046b5d)

Reply:
They do appear to work the same for a single directory but they do work differently if you are creating a tree of directories, just something to keep in mind.